README.md

```diff
@@ -0,0 +1,67 @@
+# atoum standard edition
+
+atoum standard edition is a version of atoum that directly contains code completion in IDE, configuration via a YAML file and environnement variables to simplfy the usage of atoum on CI tools. Il also comes with some usefull extensions to test protected method or to add new reports to atoum.
+
+## Install it
+
+You just have to require `atoum/standard-edition` via composer. You don't need to install atoum manually.
+
+```
+composer require atoum/standard-edition
+```
+
+## What it does
+
+* installs [atoum/atoum](https://github.com/atoum/atoum),
+* installs [atoum/config-extension](https://github,.com/jubianchi/atoum-config-extension),
```

Comment:
because like for each other extension/work around atoum, when someone start it, it's in it's own repo and then backported to atoum organisation ;)